### DISCUSSION TITLE
Matching regex for validating form

### DISCUSSION THREAD

**Discussion by enpro-github** [Budibase/budibase#4227] [Help]
I'm trying to allow a form to only allow integers or decimals: `/^\d*\.?\d+$/` . I also set m submit button to validate the form as the first step. However it is invalidating every input, including ints are floats. Am I missing something?

Thanks

**Comment by xnohat** [comment]
Step to use regex validation:

"Custom validation rules"
"Must match regex"
"Value"
regex string must without "delimiter" (/ / or ##)
